man/corosync.conf.5

```diff
@@ -788,6 +791,39 @@ potentially breaking down membership.  IPMI watchdogs are particularly
 notorious in this regard: read about kipmid_max_busy_us in IPMI.txt in
 the Linux kernel documentation.
 
+
+.PP
+Within the
+.B nozzle
+directive it is possible to specify options for a libnozzle device. This is a pseudo
+ethernet device that routes network traffic through a channel on the corosync knet network
```

Comment:
There should be something like "unreliable, unordered channel". I'm not entirely sure how to formulate short version of it, but idea is to be double safe.